src/smart-components/group/groups.scss

```diff
@@ -1,3 +1,20 @@
 .pf-c-table tbody tr.ins-c-rbac__group-default {
   .pf-c-table__check:first-of-type input { display: none; }
 }
+
+.pf-c-question-circle-icon {
+  color: var(--pf-global--palette--black-600);
+  margin-left: var(--pf-global--spacer--sm);
+  cursor: pointer;
+  &.icon-active {
+    color: var(--pf-global--palette--blue-400)
+  }
+}
+
+.pf-c-popover {
+  min-width: 350px;
+}
+
+body {
+  overflow: hidden
```

Comment:
I don't like this rule. It might mess up the whole layout. Can you add the overflow to a different element? Is this the "fix" for messy tables with popovers?

Reply:
fixed with appendTo property, thanks for help @Hyperkid123